.github/workflows/ci.yml

```
@@ -9,38 +9,27 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.7"]
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
```

`dask/dask` and `dask/distributed` is only supporting 3.9+, I suggest to drop 3.8 right away
```suggestion
        python-version: ["3.9", "3.10", "3.11"]
```